ImageKitSample/Program.cs

```diff
@@ -0,0 +1,410 @@
+                OverlayImage("/folder/file.jpg"). // leading slash case
+                OverlayImageTrim(false).
+                OverlayImageAspectRatio("4:3").
+                OverlayImageBackground("0F0F0F").
+                OverlayImageBorder("10_0F0F0F").
+                OverlayImageDpr(2).
+                OverlayImageQuality(50).
+                OverlayImageCropping("force").
+                OverlayText("two words").
+                OverlayTextFontSize(20).
+                OverlayTextFontFamily("Open Sans").
+                OverlayTextColor("00FFFF").
+                OverlayTextTransparency(5).
+                OverlayTextTypography("b").
+                OverlayBackground("00AAFF55").
+                OverlayTextEncoded("b3ZlcmxheSBtYWRlIGVhc3k%3D").
+                OverlayTextWidth(50).
+                OverlayTextBackground("00AAFF55").
+                OverlayTextPadding(40).
+                OverlayTextInnerAlignment("left").
+                OverlayRadius(10).
+                Progressive(true).
+                Lossless(true).
+                Trim(5).
+                Metadata(true).
+                ColorProfile(true).
+                DefaultImage("folder/file.jpg/"). //trailing slash case
+                Dpr(3).
+                EffectSharpen(10).
+                EffectUsm("2-2-0.8-0.024").
+                EffectContrast(true).
+                EffectGray().
+                Original().
+                Raw("h-200).w-300).l-image).i-logo.png).l-end")
+                ;
+
+            string imageURL = imagekit.Url(trans).Path(path).TransformationPosition("query").Generate();			
+           
+            Console.WriteLine("Url for first image transformed with height: 300, width: 400 - {0}", imageURL);
+
+            ///// Generating Signed URL
+            var imgURL1 = "https://ik.imagekit.io/demo/default-image.jpg";
+            string[] queryParams = { "b=123", "a=test" };
+            try
+            {
+                var signedUrl = imagekit.Url(new Transformation().Width(400).Height(300))
+                .Src(imgURL1)
+                .QueryParameters(queryParams)
+                .ExpireSeconds(600)
+                .Signed()
+                .Generate();
+                Console.WriteLine("Signed Url for first image transformed with height: 300, width: 400: - {0}", signedUrl);
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine(ex.Message);
+            }
+            #endregion
+
+            #region  Upload BY URI| Bytes | Base64
+
+            // Upload By URI
+            FileCreateRequest request = new FileCreateRequest
+            {
+                Url = new Uri(@"http://www.google.com/images/logos/ps_logo2.png"),
+                FileName = "test.jpg",
+            };
+            Result resp1 = imagekit.Upload(request);
+
+            // Upload by bytes
+            byte[] bytes = System.IO.File.ReadAllBytes(@"image file path");
```

Comment:
`@"image file path"` won't work. Please give the path of the test.jpg file you have.